grafana/Tiltfile

```diff
@@ -138,9 +138,10 @@ def grafana(context, plugin_files, grafana_image='grafana/grafana', grafana_vers
     chart_values['image']['repository'] = image_repository
     chart_values['image']['tag'] = image_tag
 
-    # Sets the Grafana Enterprise License Text
+    # Sets the Grafana Enterprise License Text and enables the accessControlOnCall feature flag
     chart_values['grafana.ini']['enterprise'] = {}
     chart_values['grafana.ini']['enterprise']['license_text'] = license
+    chart_values['grafana.ini']['feature_toggles']['enable'] = 'accessControlOnCall'
```

Comment:
I was wondering if we want to add these as a setting or config value, so that each user of tilt-extensions can supply their own set of feature flags?